BraintreeDropIn/BTDropInController.m

```diff
@@ -15,6 +15,11 @@
 #import <BraintreeCard/BraintreeCard.h>
 #import <BraintreeUnionPay/BraintreeUnionPay.h>
 #endif
+#if __has_include("BraintreePaymentFlow.h")
+#import "BraintreePaymentFlow.h"
+#else
+#import <BraintreePaymentFlow/BraintreePaymentFlow.h>
+#endif
```

Comment:
Why do we do this `#if __has_include` dance? I know it's all over the place, but I'm not totally clear on what it's doing.

Reply:
It supports the framework or module import integration styles.